src/test/scala/mesosphere/marathon/upgrade/ScalingPropositionTest.scala

```diff
@@ -0,0 +1,124 @@
+package mesosphere.marathon.upgrade
+
+import mesosphere.marathon.Protos.MarathonTask
+import mesosphere.marathon.state.Timestamp
+import mesosphere.marathon.tasks.MarathonTasks
+import org.scalatest.{ FunSuite, Matchers }
+
+class ScalingPropositionTest extends FunSuite with Matchers {
+
+  test("propose - empty tasksToKill should lead to ScalingProposition(None, _)") {
+    val running = Set.empty[MarathonTask]
+    val toKill = Some(Set.empty[MarathonTask])
+    val scaleTo = 0
+    val proposition = ScalingProposition.propose(running, toKill, scaleTo)
```

Comment:
No need to change but an alternative for separate variable here might actually be named arguments.